wp-content/themes/core/assets/css/src/theme/forms/_mixins.pcss

```diff
@@ -11,8 +11,8 @@
  * ----------------------------------------------------------------------------- */
 
 @define-mixin form-control-group-radio-checkbox {
-	padding-top: var(--form-control-group-radio-checkbox-padding-top);
-	padding-bottom: var(--form-control-group-radio-checkbox-padding-bottom);
+	padding-top: 8px;
```

Comment:
We have spacing vars available (at least for 8px, `--spacer-10`). I think it might be worth it to use that?

Reply:
absolutely, hadn't made a pass looking for where we can use vars yet but that's on my list!